docs/advanced/naming.md

```diff
@@ -1,190 +1,245 @@
-[Group] Anime Name - 18.5 [BD 1080p FLAC] [CRC].mkv
-[Group] Anime Name - 06.5 (SP) [BD 1080p FLAC].mkv
-[Group] Anime Name - Title of the Episode [CRC].mkv
-[Group] Anime Name - 17.5 (OVA) [CRC].mkv
-```
+![Releases using good naming schemes](/static/advanced/naming/releases-good-naming.png)
 
-Good names:
+### Recommended Schemes
 
-```
-Anime Name - 06.5 - S00E04 - Title of the Episode (BD 1080p HEVC FLAC) [Dual Audio] [CRC32] [Group].mkv
-Anime Name - S00E01 - Title of the Episode (AMZN WEB-DL 1080p H.264 EAC3) [Dual Audio] [Group].mkv
-Anime Name - 18.5 - S00E05 - Title of the Episode (BD 1080p HEVC FLAC) [Dual Audio]-Group.mkv
-Anime Name (2022) - S00E01 - Title of the Episode (BD Remux 1080p HEVC FLAC) [Dual Audio]-Group.mkv
-Anime.Name.S00E09.Title.of.the.Episode.1080p.BluRay.Opus2.0.x264-Hi10P-Group.mkv
-```
+A good file name allows a viewer to easily understand what the file is and what it contains. See the recommended variants below:
+
++++ :icon-file-code: Variant 1
+
+![Example file name using variant 1](/static/advanced/naming/variant-1.png)
+
+!!!warning
+Spaces and special characters (i.e. apostrophes `'`, commas `,`) should be replaced with periods `.` instead.
+!!!
+
+![#ed8796](https://placehold.co/14x14/ed8796/ed8796.png) Show title
+:   The full show/series or movie title. *We recommend using names found in databases such as [TheTVDB](https://www.thetvdb.com) (TV series) and [The Movie Database (TMDB)](https://www.themoviedb.org/?language=en-US) (movies). This can be in English or romanized Japanese*
+
+    !!!warning
+    If there are multiple versions of the series, you must include the year within the title (e.g. [`Hunter.x.Hunter.1999`](https://thetvdb.com/series/hunter-x-hunter) vs. [`Hunter.x.Hunter.2011`](https://thetvdb.com/series/hunter-x-hunter-2011)). *For movies, this should always be included*
+    !!!
+
+![#f5a97f](https://placehold.co/14x14/f5a97f/f5a97f.png) Season/episode number
+:   The season and episode number
+
+    !!!info
+    For specials, use `S00EXX` for guaranteed automatic parsing and correct identification. *However, it is suggested to add an additional identifier such as including `OVA` or the episode title (i.e. `S00E01.OVA.Title`) to easily differentiate between episodes*
+    !!!
+
+![#eed49f](https://placehold.co/14x14/eed49f/eed49f.png) Episode title [!badge variant="success" text="Optional"]
+:   The title of the episode. *We recommend using names found in databases such as [TheTVDB](https://www.thetvdb.com) (TV series) and [The Movie Database (TMDB)](https://www.themoviedb.org/?language=en-US) (movies)*
+
+    !!!warning
+    This should generally be skipped to avoid creating file paths larger than 256 characters, which may cause problems with various programs.
+    !!!
+
+![#a6da95](https://placehold.co/14x14/a6da95/a6da95.png) Version number [!badge variant="warning" text="Conditional"]
+:   The version number for patched releases in `REPACKX` format, where `X` is the version number
```

Comment:
slightly wrong. v2 would be `REPACK`, v3 would be `REPACK2`, v4 would be `REPACK3` and so on.